### PR TITLE
SumoLogic with fetch-incidents context and HumanReadable

### PR DESCRIPTION
**Pull request by slavikm**
Re-implementation of SumoLogic in BYOI that supports fetch (with mapping), human readable output and context.

@aviadl please notice that the output is a bit different from original because original looked unusable to me.

### CODE REVIEW

**Comment by slavikm** [comment]
API documentation - https://help.sumologic.com/APIs/Search-Job-API/About-the-Search-Job-API

File: Integrations/integration-SumoLogic.yml [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+            var s = search(args.query, args.from, args.to, a2i(args.limit, defaultLimit), a2i(args.offset, 0), args.timezone,
+                a2i(args.maxTimeToWaitForResults, defaultSearchTimeout) * 60, a2i(params.sleepBetweenChecks, defaultSleep));
+            var md = '';
+            var ec = {};
+            if (s.messages && s.messages.length > 0) {
+                md = tableToMarkdown('Messages', s.messages) + '\n';
+                ec.Search = {Messages: s.messages.length > defaultLimit ? s.messages.slice(0, defaultLimit) : s.messages};
+            }
+            if (s.records && s.records.length > 0) {
+                md += tableToMarkdown('Records', s.records);
```
Comment (aviadl) [inline Integrations/integration-SumoLogic.yml]: Sumlogic Records ?
And same for Messages ?

File: Integrations/integration-SumoLogic.yml [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+            setLastRun({time: now});
+            return JSON.stringify(incidents);
+        case 'search':
+            var s = search(args.query, args.from, args.to, a2i(args.limit, defaultLimit), a2i(args.offset, 0), args.timezone,
+                a2i(args.maxTimeToWaitForResults, defaultSearchTimeout) * 60, a2i(params.sleepBetweenChecks, defaultSleep));
+            var md = '';
+            var ec = {};
+            if (s.messages && s.messages.length > 0) {
+                md = tableToMarkdown('Messages', s.messages) + '\n';
+                ec.Search = {Messages: s.messages.length > defaultLimit ? s.messages.slice(0, defaultLimit) : s.messages};
```
Comment (aviadl) [inline Integrations/integration-SumoLogic.yml]: If i will run the same command twice in war room what will happen to the context ? will it override or duplicate ?

Comment (slavikm) [inline Integrations/integration-SumoLogic.yml]: It will add to the context

File: Integrations/integration-SumoLogic.yml (hunk 1 above)
Comment (slavikm) [inline Integrations/integration-SumoLogic.yml]: I guess I can add

**Pull request merged**